docs/kits/Business Partner Kit/Documentation BPDM/page_use-cases.md

```diff
@@ -0,0 +1,2294 @@
+                        "value": "63485ec63cd3b21131fb69ac",
+                        "type": {
+                            "technicalKey": "CDQID",
+                            "name": "CDQ Identifier",
+                            "url": ""
+                        },
+                        "issuingBody": {
+                            "technicalKey": "CDQ",
+                            "name": "CDQ AG",
+                            "url": ""
+                        },
+                        "status": {
+                            "technicalKey": "CDQ_IMPORTED",
+                            "name": "Imported from CDQ but not synchronized"
+                        }
+                    }
+                ],
+                "names": [],
+                "legalForm": null,
+                "status": null,
+                "profileClassifications": [],
+                "types": [],
+                "bankAccounts": [],
+                "roles": [],
+                "relations": [],
+                "currentness": "2022-10-14T15:00:06.325509Z"
+            }
+        },
+        {
+            "score": 0.0,
+            "legalEntity": {
+                "bpn": "BPNL00000006MF69",
+                "identifiers": [
+                    {
+                        "value": "63485ec63cd3b21131fb69ae",
+                        "type": {
+                            "technicalKey": "CDQID",
+                            "name": "CDQ Identifier",
+                            "url": ""
+                        },
+                        "issuingBody": {
+                            "technicalKey": "CDQ",
+                            "name": "CDQ AG",
+                            "url": ""
+                        },
+                        "status": {
+                            "technicalKey": "CDQ_IMPORTED",
+                            "name": "Imported from CDQ but not synchronized"
+                        }
+                    }
+                ],
+                "names": [],
+                "legalForm": null,
+                "status": null,
+                "profileClassifications": [],
+                "types": [],
+                "bankAccounts": [],
+                "roles": [],
+                "relations": [],
+                "currentness": "2022-10-14T15:00:06.325574Z"
+            }
+        }
+    ]
+}
+```
+
+### 1.1 Direct response
+
+Scenario: The user enters a valid EU_VAT_ID_DE and the internal API returns information about only one legal entity.
+
+Flow:
+
+* The user inputs a valid EU_VAT_ID_DE to the Legal Entity Controller.
+* The Legal Entity Controller sends an API call to the internal API with the EU_VAT_ID_DE.
+* The internal API returns information about only one legal entity.
+* The Legal Entity Controller displays the information about the legal entity to the user.
+* Once you have leagal enity shown then direct response you can get using idType as BPN value as shown in below request and response.
```

Comment:
Flow or steps are good but internal system steps should be omitted. The steps should be from the perspective of the user. What does the user need to do and what does he see as a (intermediate) result.